src/connectedvmware/azext_connectedvmware/_params.py

```diff
@@ -346,12 +352,77 @@ def load_arguments(self, _):
         c.argument(
             'vm_name', options_list=['--vm-name'], help="Name of the VM.",
         )
+
+    with self.argument_context('connectedvmware vm extension list') as c:
+        c.argument('vm_name', options_list=['--vm-name'], type=str, help='The name of the vm containing the extension.')
+        c.argument('expand', type=str, help='The expand expression to apply on the operation.')
+
+    with self.argument_context('connectedvmware vm extension show') as c:
+        c.argument(
+            'vm_name', options_list=['--vm-name'], type=str, help='The name of the vm containing the extension.',
+            id_part='name')
+        c.argument('name', type=str, help='The name of the vm extension.', id_part='child_name_1')
+
+    with self.argument_context('connectedvmware vm extension create') as c:
+        c.argument(
+            'vm_name', options_list=['--vm-name'], type=str, help='The name of the vm where the extension '
+            'should be created or updated.')
+        c.argument('name', type=str, help='The name of the vm extension.')
+        c.argument('tags', tags_type)
+        c.argument('force_update_tag', type=str, help='How the extension handler should be forced to update even if '
+                   'the extension configuration has not changed.')
+        c.argument('publisher', type=str, help='The name of the extension handler publisher.')
+        c.argument('type_', options_list=['--type'], type=str, help='Specifies the type of the extension; an example '
+                   'is "CustomScriptExtension".')
+        c.argument('type_handler_version', type=str, help='Specifies the version of the script handler.')
+        c.argument('auto_upgrade_minor', arg_type=get_three_state_flag(), help='Indicate whether the extension should '
+                   'use a newer minor version if one is available at deployment time. Once deployed, however, the '
+                   'extension will not upgrade minor versions unless redeployed, even with this property set to true.')
+        c.argument('settings', type=validate_file_or_dict, help='Json formatted public settings for the extension. '
+                   'Expected value: json-string/json-file/@json-file.')
+        c.argument('protected_settings', type=validate_file_or_dict, help='The extension can contain either '
+                   'protectedSettings or protectedSettingsFromKeyVault or no protected settings at all. Expected '
+                   'value: json-string/json-file/@json-file.')
+        c.argument('instance_view_type', type=str, help='Specify the type of the extension; an example is '
+                   '"CustomScriptExtension".', arg_group='Instance View')
+        c.argument('inst_handler_version', type=str, help='Specify the version of the script handler.',
+                   arg_group='Instance View')
+
+    with self.argument_context('connectedvmware vm extension update') as c:
```

Comment:
It seems that it and `connectedvmware vm extension create` have several common parameters. Could we consider using for loop to share them?
Such as:
```Python
for scope in ['connectedvmware vm extension update', 'connectedvmware vm extension create']:
        with self.argument_context(scope) as c:
```

Reply:
Fixed